usr/sbin/regionsrv-enabler-azure

```diff
@@ -12,6 +12,40 @@
 # Lesser General Public License for more details.
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library.
+
+"""
+Check the different states of the license type:
+START -> The state of the instance when this code is executed
+         This code is executed by a timer
+
+    START   LICENSE CHANGED TO     CACHE   ACTION
+1   BYOS-1  PAYG                   No      Register - Update Infra
+2   BYOS-1  PAYG                   BYOS    Register - Update Infra
+3   BYOS-1  BYOS-1                 No      Do nothing
+4   BYOS-1  BYOS-1                 BYOS    Do nothing
+5   BYOS-1  BYOS-2                 No      Do nothing
+6   BYOS-1  BYOS-2                 BYOS    Do nothing
```

Comment:
Case 5 & 6 collapse to cases 3 and 4 because the customer cannot change the licence via AHB to a registered state. The action is always

- Change license to BYOS, this is always BYOS-1 to BYOS-1
- Register, this would then result in BYOS-2 or BYOS-3

Given the table we have this is hard to express. At the same time adding more columns such as "Additional user action" is not conducive to making it more readable. I suggest we  make entries 3 & 4 look like this:

`BYOS-1  BYOS-1  (BYOS_2, BYOS-3)* .....`

and the add a footnote like so

`* The user has manually registered he instance with their registration key`
-